node/pkg/p2p/cutover.go

```diff
@@ -0,0 +1,138 @@
+package p2p
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"go.uber.org/zap"
+)
+
+// The format of this time is very picky. Please use the exact format specified by cutOverFmtStr!
+const cutOverTimeStr = "2024-12-31T23:59:59-0000"
```

Comment:
can we create separate cutover strings per environment? that way we can set testnet to something in the near future and leave mainnet `""` until the testnet cutover completes successfully. this will be necessary to support testnet and mainnet guardians and spies with the same codebase concurrently.